dev-docs/modules/userId.md

```diff
@@ -323,6 +323,47 @@ pbjs.setConfig({
 });
 {% endhighlight %}
 
+### DMD ID by DMD Marketing Corp
+
+DMD is the preeminent supplier of US-based healthcare professional (HCP) identity data to the pharmaceutical, health system and medical publishing industries. DMD is the only data provider that has acquired its deterministic identity data through a fully consented, first-party, opt-in process. DMD’s privacy policy that can be found at [Privacy Policy](https://hcn.health/privacy-policy). 
+
+For assistance setting up your module, please contact us at prebid@dmdconnects.com
+
+Add the DMD ID to your Prebid.js Package with:
+
+{: .alert.alert-info :}
+gulp build --modules=userId,dmdIdSystem
+
+#### DMD ID Registration
+
+Please reach out to [prebid@dmdconnects.com](mailto:prebid@dmdconnects.com) to request your `api_key`
+
+#### DMD ID Configuration
+
+{: .table .table-bordered .table-striped }
+| Param under userSync.userIds[] | Scope | Type | Description | Example |
+| --- | --- | --- | --- | --- |
+| name | Required | String | The name of Module | `"dmdId"` |
+| storage | Required | Object |  |
+| storage.name | Required | String | `dmd-dgid` |
+| params | Required | Object | Container of all module params. |  |
+| params | Required | Object | Container of all module params. |  |
```

Comment:
i think this is an accidental duplicate line

Reply:
@smenzer Removed duplicate line and updated PR

Reply:
@smenzer is there anything else to add in the docs?

Reply:
looks ok to me